test/regress/regress1/arith/arith-brab-test.smt2

```diff
@@ -0,0 +1,21 @@
+; EXPECT: sat
```

Comment:
Will need to additionally add `; COMMAND-LINE: --arith-brab` before the `; EXPECT` line.